components/colorslider/index.css

```
@@ -45,6 +45,9 @@
 			display: none;
 		}
 	}
+
+	/* Settings for nested Color handle */
```

```suggestion
	/* @passthroughs -- settings for nested color handle */
```

You can also add the passthroughs flag here to indicate these properties are being pushed down to a nested component. We use this in our stylelint plugins that report on whether custom properties are defined or unused.